packages/interpretations/src/components/details/SharingText.js

```diff
@@ -0,0 +1,20 @@
+import i18n from '@dhis2/d2-i18n';
```

Comment:
As far as I can tell, this is not a component, but just returns some text, right?  Then the file should be camelCase:
sharingText.js

Reply:
Yes that is correct, it only returns text - I've changed the name of the text file to use camelCase instead of PascalCase